transactions.py

```diff
@@ -0,0 +1,66 @@
+import datetime
+import json
+
+transactions = {}
+transaction_counter = 1
+
+TRANSACTION_MENU = """
+1. Add transaction
+2. Read ALL transactions
+3. Read one transaction by transaction ID
+x. Exit to Main Menu
+"""
+
+
+def read_transactions():
+    try:
+        with open('transactions_log.json', 'r') as file:
+            data = json.load(file)
+        return data
+    except (FileNotFoundError, json.decoder.JSONDecodeError):
+        data = []
+    return data
+
+
+def read_transaction(transaction_id):
+    try:
+        with open('transactions_log.json', 'r') as file:
+            data = {}
+            data = json.load(file)
+            for transaction in data:
+                if transaction.get('transaction_id') == transaction_id:
+                    return transaction
+            print("The selected transaction doesn't exist")
+    except (FileNotFoundError, json.decoder.JSONDecodeError):
+        data = []
+    return data
+
+
+def write_transactions(data):
+    with open('transactions_log.json', 'w') as file:
+        json.dump(data, file, indent=4)
+
+
+def add_transaction():
+    data = list(read_transactions())
```

Comment:
Reading this every time is time-consuming. We can use a variable that holds all transactions. First thing we do is read the transactions, second, whenever we update something to the transactions we update the variable and the file (in case of a crash)